### PR TITLE
feat: Add support for Google Login

### PR DESCRIPTION
**Pull request by sasjo**
This pull request introduces Google Login to limit access to the tech-radar. The login is pinned to extendaretail.com. A new backend server has been introduced to validate JWTs on the server-side. The application validates that the token is a valid Google token and that the hosted domain in the JWT is extendaretail.com.

With this change, we can run the radar on a publicly available URL as is the case with Cloud Run.

### CODE REVIEW

**Comment by sasjo** [comment]
Related to #83.

File: package.json [hunk 1]
```diff
@@ -5,9 +5,13 @@
   "description": "Extenda Retail Technology Radar",
   "main": "src/js/index.jsx",
   "scripts": {
-    "build": "webpack-cli --mode production",
+    "build": "npm run build:frontend && npm run build:backend",
+    "build:frontend": "webpack-cli --mode production",
+    "build:backend": "npm run build --prefix server",
     "lint:js": "eslint src test",
     "start": "webpack-dev-server --open --watch --mode development --output-path build",
+    "backend": "PORT=3000 npm start --prefix server",
```
Comment (vadimb) [inline package.json]: This env wont be portable(Windows anybody?) so a way will be to do `npm run env PORT=3000 -- npm start --prefix server`

Comment (sasjo) [reply package.json]: Thanks, didn't know about `npm run env`!

File: package.json [hunk 2]
```diff
@@ -10,7 +10,7 @@
     "build:backend": "npm run build --prefix server",
     "lint:js": "eslint src test",
     "start": "webpack-dev-server --open --watch --mode development --output-path build",
-    "backend": "PORT=3000 npm start --prefix server",
+    "backend": "npm start --prefix server",
     "auth": "BACKEND=1 webpack-dev-server --open --watch --mode production --host local.extendaretail.com --output-path build",
```
Comment (vadimb) [inline package.json]: Sorry, `BACKEND` can be changed.

Comment (sasjo) [reply package.json]: Ah, thanks!

**Pull request merged**